src/containers/Board/SymbolDetails/SymbolDetails.js

```diff
@@ -25,7 +25,7 @@ export class SymbolDetails extends Component {
       id: shortid.generate(), // todo: not here
       type: 'symbol',
       label: '',
-      text: '',
+      description: '',
```

Comment:
‘description’ symbol property will later be changed to ‘vocalization’ to align with OBF. Issue #35

Reply:
To clarify, don't change this now? If so, all changes have been made.

Reply:
that'll be done with issue #35